lib/src/widgets/animated_content_widget.dart

```diff
@@ -24,16 +27,22 @@ class AnimatedContentWidget extends StatelessWidget {
   /// The delegate responsible for building the animated content.
   final AnimatedContentDelegate delegate;
 
+  /// The border radius of the book.
+  final BorderRadius borderRadius;
```

Comment:
could we have a default value for this?

Reply:
oh, you are right, this.borderRadius = BorderRadius.zero, will be fine right?

Reply:
done